subsys/trusted_storage/Kconfig

```diff
@@ -0,0 +1,195 @@
+# Trusted storage subsystem configuration options
+
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig TRUSTED_STORAGE
```

Comment:
I suggested we namespace this under PSA.

I can't see that we are adding a generic interface using PSA ITS as the backend, so I don't think there is any reason to avoid using PSA here.